session-storage/check.js

```diff
@@ -1,11 +1,41 @@
 /* eslint-disable */
-let value = '';
+let value = "";
+
+const btn = document.getElementsByClassName("btn")[0];
+
+const updateLiveData = () => {
+  const liveData = document.getElementById("live-data");
+  liveData.innerHTML = '';
+  const entries = Object.entries(sessionStorage);
+  entries.forEach(entry => {
+    const newLi = document.createElement("li");
+    newLi.innerHTML = `${entry[0]}: ${entry[1]} 
+    <button onclick="deleteEntry('${entry[0]}')">x</button>`
+    liveData.appendChild(newLi);
+  });
+};
+
+const deleteEntry = (key) => {
+  sessionStorage.removeItem(key);
+  updateLiveData();
+}
+
+btn.onclick = e => {
+  e.preventDefault();
+  const key = document.getElementById("key").value;
+  value = document.getElementById("value").value;
+  sessionStorage.setItem(key, value);
+
+  updateLiveData();
```

Comment:
How about adding a new item only in dom, instead of loading all data on every update?